### PR TITLE
Ensure slovak localization terms match their base terms

### PR DESCRIPTION
**Pull request by mskrip**

### CODE REVIEW

File: settings/l10n/sk.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ OC.L10N.register(
     "Backend doesn't support password change, but the user's encryption key was updated." : "Backend nepodporuje zmenu hesla, ale šifrovací kľúč používateľa bol zmenený.",
     "installing and updating apps via the app store or Federated Cloud Sharing" : "Inštalovať a aktualizovať aplikácia pomocou obchodu alebo Zdieľaného Cloudového Úložiska",
     "Federated Cloud Sharing" : "Sprístupnenie prostredníctvom Federated Cloud",
-    "cURL is using an outdated %s version (%s). Please update your operating system or features such as %s will not work reliably." : "cURL používa zastaralú %sverziu (%s).  Prosím aktualizujte si operačný systém pretože %snebude fungovať spoľahlivo.",
+    "cURL is using an outdated %s version (%s). Please update your operating system or features such as %s will not work reliably." : "cURL používa zastaralú %s verziu (%s).  Prosím aktualizujte si operačný systém pretože %s nebude fungovať spoľahlivo.",
```
Comment (lubo) [inline settings/l10n/sk.js]: There's duplicate space between `(%s).`  and `Prosím`.

File: settings/l10n/sk.js [hunk 2]
```diff
@@ -226,7 +226,7 @@ OC.L10N.register(
     "Cancel" : "Zrušiť",
     "Choose as profile picture" : "Použiť ako obrázok avatara",
     "Details" : "Detaily",
-    "You are using <strong>%s</strong> of <strong>%s</strong> (<strong>%s %%</strong>)" : "Využívate <strong>%s</strong> z <strong>%s</strong>(<strong>%s%%</strong>)",
+    "You are using <strong>%s</strong> of <strong>%s</strong> (<strong>%s %%</strong>)" : "Využívate <strong>%s</strong> z <strong>%s</strong>(<strong>%s %%</strong>)",
```
Comment (lubo) [inline settings/l10n/sk.js]: Missing space before parentheses.

File: settings/l10n/sk.js (hunk 1 above)
Comment (mskrip) [reply settings/l10n/sk.js]: Removed this duplicate spaces and many more I found

File: settings/l10n/sk.js (hunk 2 above)
Comment (mskrip) [reply settings/l10n/sk.js]: Fixed

File: settings/l10n/sk.js [hunk 3]
```diff
@@ -37,16 +37,16 @@ OC.L10N.register(
...
-    "Password for %1$s changed on %2$s" : "Heslo pre %1$sbolo zmenené na %2$s",
+    "%1$s changed your password on %2$s." : "%1$s zmenil Vaše heslo na %2$s",
+    "Your password on %s was changed." : "Vaše heslo na %s bolo zmenené.",
+    "Your password on %s was reset by an administrator." : "Vaše heslo na %s bolo resetované administrátorom.",
+    "Password for %1$s changed on %2$s" : "Heslo pre %1$s bolo zmenené na %2$s",
     "Password changed for %s" : "Heslo zmenené pre %s",
     "If you did not request this, please contact an administrator." : "Ak ste to nevyžadovali, prosím kontaktujte administrátora.",
-    "%1$s changed your email address on %2$s." : "%1$szmenil Vašu emailovú adresu na %2$s",
-    "Your email address on %s was changed." : "Vaša emailová adresa na %sbola zmenená.",
+    "%1$s changed your email address on %2$s." : "%1$s zmenil Vašu emailovú adresu na %2$s",
```
Comment (lubo) [inline settings/l10n/sk.js]: Missing interpunction.

File: settings/l10n/sk.js [hunk 4]
```diff
@@ -37,16 +37,16 @@ OC.L10N.register(
     "Unable to change email address" : "Nemožno zmeniť emailovú adresu",
     "In order to verify your Twitter account, post the following tweet on Twitter (please make sure to post it without any line breaks):" : "Ak chcete overiť svoj Twitter účet, napíšte nasledujúci tweet (uistite sa, že ho zasielate bez zalomenia riadkov):",
     "In order to verify your Website, store the following content in your web-root at '.well-known/CloudIdVerificationCode.txt' (please make sure that the complete text is in one line):" : "Pre verifikáciu vašich webových stránok uložte nasledujúci obsah v koreňovom adresári webovej stránky v umiestnení '.well-known/CloudIdVerificationCode.txt' (uistite sa, že bol text vložený ako jediný riadok)",
-    "%1$s changed your password on %2$s." : "%1$szmenil Vaše heslo na %2$s",
-    "Your password on %s was changed." : "Vaše heslo na %sbolo zmenené.",
-    "Your password on %s was reset by an administrator." : "Vaše heslo na %sbolo resetované administrátorom.",
-    "Password for %1$s changed on %2$s" : "Heslo pre %1$sbolo zmenené na %2$s",
+    "%1$s changed your password on %2$s." : "%1$s zmenil Vaše heslo na %2$s",
```
Comment (lubo) [inline settings/l10n/sk.js]: Missing interpunction.

**Comment by lubo** [comment]
Go for upstream.

**Pull request closed without merging**